### PR TITLE
50149_Category_Index

### PR DESCRIPTION
**Pull request by noor-framgia**

### CODE REVIEW

**Comment by AhsanBracu** [comment]
ok.

**Comment by SSalekin** [comment]
ok.

**Comment by manazirahsan** [comment]
ok

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Admin::CategoriesController < ApplicationController
+
+  def index
+    @categories = Category.all
```
Comment (dieunb) [inline app/controllers/admin/categories_controller.rb]: cancancan?

File: config/routes.rb [hunk 2]
```diff
@@ -7,6 +9,9 @@
     get "contact"=> "static_pages#contact"
 
     resources :requests, only: [:new, :create, :index]
-    devise_for :users, controllers: {registrations: "registrations"}
+
+    namespace :admin do
```
Comment (dieunb) [inline config/routes.rb]: admin root page

File: app/controllers/admin/categories_controller.rb (hunk 1 above)
Comment (noor-framgia) [inline app/controllers/admin/categories_controller.rb]: chi Rownak  will implement it

File: config/routes.rb (hunk 2 above)
Comment (noor-framgia) [inline config/routes.rb]: can not understand

Comment (dieunb) [inline config/routes.rb]: should set admin root page, same `statis_pages#home`

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
\ No newline at end of file
+
+Category.create name: "Drama"
```
Comment (dieunb) [inline db/seeds.rb]: use `faker` and `fabrication` gem to make sample data, next pull

Comment (noor-framgia) [inline db/seeds.rb]: ok. i'll definitely :)

**Pull request merged**